app/experimenter/experiments/migrations/0132_nimbus_add_primary_secondary_probe_sets.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.0.7 on 2020-10-30 21:14
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
```

Comment:
Oh, fold these two migrations together, just delete them both and then run `makemigrations` again.  Generally there's no reason to have more than one migration per pr.